modules/identity-provider-htpasswd-secret.adoc

```diff
@@ -0,0 +1,44 @@
+// Module included in the following assemblies:
+//
+// * authentication/identity_providers/configuring-htpasswd-identity-provider.adoc
+
+[id='identity-provider-creating-htpasswd-secret-{context}']
+= Creating the HTPasswd Secret
+
+To use the HTPasswd identity provider, you must define a secret containing the
```

Comment:
s/containing/that contains

Reply:
I've incorporated this suggestion into all of the *secret files.